bottles/frontend/views/preferences.py

```diff
@@ -149,14 +153,24 @@ def set_path(_dialog, response, _file_dialog):
                     _("Choose where to store the new bottles (this will not move the existing ones)."))
             _file_dialog.destroy()
 
-        FileChooser(
-            parent=self.window,
-            title=_("Choose new bottles path"),
-            action=Gtk.FileChooserAction.SELECT_FOLDER,
-            buttons=(_("Cancel"), _("Select")),
-            native=False,
-            callback=set_path
-        )
+        if "FLATPAK_ID" in os.environ:
+            FileChooser(
+                parent=self.window,
+                title=_("Choose new bottles path"),
+                action=Gtk.FileChooserAction.SELECT_FOLDER,
+                buttons=(_("Cancel"), _("Select")),
+                callback=set_path,
+                native=True
+            )
+        else:
+            FileChooser(
+                parent=self.window,
+                title=_("Choose new bottles path"),
+                action=Gtk.FileChooserAction.SELECT_FOLDER,
+                buttons=(_("Cancel"), _("Select")),
+                callback=set_path,
+                native=False
+            )
```

Comment:
```suggestion
        if "FLATPAK_ID" in os.environ:
            use_portal = True
        else:
            use_portal = False

        FileChooser(
            parent=self.window,
            title=_("Choose new Bottles path"),
            action=Gtk.FileChooserAction.SELECT_FOLDER,
            buttons=(_("Cancel"), _("Select")),
            callback=set_path,
            native=use_portal
        )
```

Reply:
`native="FLATPAK_ID" in os.environ`

Reply:
oh lol thats cool

Reply:
This check in general isn't really needed, portals can be used on non flatpak apps, there isn't really any point in making them run only in the flatpak